### ISSUE TITLE
Protractor e2e test hang with ngx-scrollbar

### ISSUE THREAD

**Issue by manhquang144** [MurhafSousli/ngx-scrollbar#388]
I am replacing my angular app scrollbar with ngx-scrollbar, everything is working fine but my e2e tests start to timeout weirdly. 
Have anyone faced similar issues before or know how to fix this issue?
Thank you a lot

#### Reproduction
I have create a simple angular project with two e2e tests: normalist.e2e-spec.ts and scrollbar.e2e-spec.ts to reproduce the issue.
Source: https://github.com/manhquang144/NgxScrollbarDemo
The two components being tested are very similar (except one is using ngx-scrollbar, the other one just has the normal browser scrollbar)

```import { AppPage } from './app.po';

describe('scrollbar page', () => {
  let page: AppPage;

  beforeEach(() => {
    page = new AppPage();
  });

  it('should display correct number of items', async () => {
    console.log('Step 1');
    await page.navigateToScrollbarList();
    console.log('Step 2');
    expect(await page.getItemCount()).toEqual(10);
    console.log('Step 3');
  });

  afterEach(() => {});
});
```

```
navigateToScrollbarList() {
    return browser.get(browser.baseUrl + '/scrollbar');
  }
```

The test with ngx-scrollbar (normalist.e2e-spec.ts) always timeout (Stop at 'Step 2') whereas the other one passes.

#### Environment

- Angular: 10
- ngx-scrollbar: 7.5.6
- Browser(s): Chrome 92.0.4515.107
- Operating System: Windows

**Comment by manhquang144** [comment]
Actually the test passes if I added changeDetection: ChangeDetectionStrategy.OnPush to the component. So I am guessing this may be related to the issue on #373 ?

**Comment by MurhafSousli** [comment]
If this is causing the issue then it should be fixed in v7.6.0, please try it out and let me know if your issue is resolved

**Comment by manhquang144** [comment]
@MurhafSousli thanks a lot for the quick fix. I can confirm that the issue is resolved with v7.6.0. 

So looks like in previous version, the protractor may wait for Angular to be "stable" but they has never been since the requestAnimationFrame keep firing.

**Comment by MurhafSousli** [comment]
Nice! cool then I am closing...
